src/App.tsx

```diff
@@ -3,33 +3,50 @@ import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-type State = {};
+type State = {
+  query: string,
+};
 
 export class App extends React.Component<{}, State> {
-  state: State = {};
+  state: State = {
+    query: '',
+  };
+
+  filterCallback = (movie: Movie) => {
+    return (
+      movie.title.toLowerCase().includes(this.state.query.toLowerCase())
+      || movie.description.toLowerCase().includes(this.state.query.toLowerCase())
+    );
```

Comment:
You can optimize this
```suggestion
  filterCallback = (movie: Movie) => {
  	const { title, description } = movie;
  	const lowerCaseQuery = query.toLowerCase();
  	
    return (
      title.toLowerCase().includes(lowerCaseQuery)
      || description.toLowerCase().includes(lowerCaseQuery)
    );
```